usersimcrs/simulator/neural/tus/tus_feature_handler.py

```diff
@@ -0,0 +1,516 @@
+"""Feature handler for the TUS simulator.
+
+For a matter of simplicity, the feature handler supports only one domain unlike
```

Comment:
Nit: "For a matter of simplicity" => "For simplicity", "only one domain" => "a single domain"

Reply:
Done.